lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlobFileSystemE2E.cs

```diff
@@ -0,0 +1,240 @@
+            const string connectionString = "";
+            var defaultContainerName = "reef-test-container-" + Guid.NewGuid();
+            _fileSystem = new AzureBlobFileSystem(connectionString);
+            _container = CloudStorageAccount.Parse(connectionString).CreateCloudBlobClient().GetContainerReference(defaultContainerName);
+            _container.CreateIfNotExists();
+        }
+
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            if (_container != null)
+            {
+                _container.DeleteIfExists();
+            }
+        }
+
+        [TestMethod]
+        public void TestDeleteE2E()
+        {
+            var blob = _container.GetBlockBlobReference(HelloFile);
+            UploadFromString(blob, "hello");
+            Assert.IsTrue(blob.Exists());
+            _fileSystem.Delete(PathToFile(HelloFile));
+            Assert.IsFalse(blob.Exists());
+        }
+
+        [TestMethod]
+        public void TestExistsE2E()
+        {
+            var helloFilePath = PathToFile(HelloFile);
+            var blob = _container.GetBlockBlobReference(HelloFile);
+            UploadFromString(blob, "hello");
+            Assert.IsTrue(_fileSystem.Exists(helloFilePath));
+            blob.DeleteIfExists();
+            Assert.IsFalse(_fileSystem.Exists(helloFilePath));
+        }
+
+        [TestMethod]
+        public void TestCopyE2E()
+        {
+            const string SrcFileName = "src";
+            const string DestFileName = "dest";
+            var srcFilePath = PathToFile(SrcFileName);
+            var destFilePath = PathToFile(DestFileName);
+            ICloudBlob srcBlob = _container.GetBlockBlobReference(SrcFileName);
+            UploadFromString(srcBlob, "hello");
+            Assert.IsTrue(srcBlob.Exists());
+            ICloudBlob destBlob = _container.GetBlockBlobReference(DestFileName);
+            Assert.IsFalse(destBlob.Exists());
+            _fileSystem.Copy(srcFilePath, destFilePath);
+            destBlob = _container.GetBlobReferenceFromServer(DestFileName);
+            Assert.IsTrue(destBlob.Exists());
+            srcBlob = _container.GetBlobReferenceFromServer(SrcFileName);
+            Assert.IsTrue(srcBlob.Exists());
+            Assert.AreEqual(_container.GetBlockBlobReference(SrcFileName).DownloadText(), _container.GetBlockBlobReference(DestFileName).DownloadText());
+        }
+
+        [TestMethod]
+        public void TestCopyToLocalE2E()
+        {
+            var helloFilePath = PathToFile(HelloFile);
+            var blob = _container.GetBlockBlobReference(HelloFile);
+            var tempFilePath = GetTempFilePath();
+            const string Text = "hello";
+            try
+            {
+                UploadFromString(blob, Text);
+                _fileSystem.CopyToLocal(helloFilePath, tempFilePath);
+                Assert.IsTrue(File.Exists(tempFilePath));
+                Assert.AreEqual(File.ReadAllText(tempFilePath), Text);
```

Comment:
Fix order of arguments Assert.AreEqual(expected: Text, actual: File.ReadAllText(tempFilePath));